src/lib/SapphireClient.ts

```diff
@@ -32,7 +32,7 @@ export class SapphireClient extends Client {
 	public logger: ILogger;
 
 	/**
-	 * The commands the framework has registered.
+	 * The commands arguments the framework has registered.
```

Comment:
Hmm, what about
```suggestion
	 * The command's arguments the framework has registered.
```

Reply:
Or, if it's not about a specific command
```suggestion
	 * The command arguments the framework has registered.
```

Reply:
it isn't - it's for all the arguments currently loaded. Though, arguments do not have to be command specific (?) so removing command entirely may be a better option. cc @kyranet @vladfrangu

Reply:
Yeah, I think it's a better option.